resources/provider/migrations/postgres/15_v0.10.0.up.sql

```diff
@@ -6867,3 +6867,8 @@ CREATE TABLE IF NOT EXISTS "aws_wafv2_web_acl_pre_process_firewall_manager_rule_
 	UNIQUE(cq_id),
 	FOREIGN KEY (web_acl_cq_id) REFERENCES aws_wafv2_web_acls(cq_id) ON DELETE CASCADE
 );
+
+
+ALTER TABLE IF EXISTS aws_efs_filesystems
+    ADD COLUMN backup_policy_status TEXT;
+
```

Comment:
just add this column to the actual create table, no need for the part in the down.